dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.eeCompatible-9.0.feature

```diff
@@ -7,3 +7,4 @@ Subsystem-Version: 9.0.0
 kind=ga
 edition=core
 WLP-Activation-Type: parallel
+WLP-Platform: jakartaee-9.0
```

Comment:
Do we want this to be 9.0 or 9.1

Reply:
Agreed it needs  to match what you put for the features...  so jakartaee-9.1

Reply:
done.   Should the file itself and the Subsystem-Version also be 9.1?